canonical-websites/www.ubuntu.com
# Use form include in existing (en) landing pages

## Done

* Use form includes in:
  * [`/engage/cloud-economics`](http://www.ubuntu.com-pr-4080.run.demo.haus/engage/cloud-economics)
  * [`/engage/fingbox-case-study`](http://www.ubuntu.com-pr-4080.run.demo.haus/engage/fingbox-case-study)
  * [`/engage/iot-business-model`](http://www.ubuntu.com-pr-4080.run.demo.haus/engage/iot-business-model)

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: [http://0.0.0.0:8001/](http://0.0.0.0:8001/)
- Run through the following [QA steps](https://github.com/canonical-webteam/practices/blob/master/workflow/qa-steps.md)
- Ensure the form is present and works (taking you to the right download) on each of these pages
